infrastructure/docker/build/Dockerfile-cache-config

```diff
@@ -37,9 +37,14 @@ RUN if [[ ${RHEL_VERSION%%.*} -ge 8 ]]; then \
 		git \
 		rpm-build \
 		rsync \
+		gzip \
 		epel-release && \
 	yum -y clean all
 
+RUN yum -y install dnf-plugins-core
+RUN yum -y config-manager --set-enabled PowerTools || yum -y config-manager --set-enabled powertools
```

Comment:
> ```shell
> yum -y config-manager --set-enabled PowerTools
> ```

Your call, but IMO no need to support CentOS 8 before 8.3.

Reply:
I'd prefer to leave it, if you don't mind. Better to support all minor versions where possible, and it's easy to do here.